python/src/scipp/__init__.py

```diff
@@ -110,14 +110,18 @@ def custom_formatwarning(msg, *args, **kwargs):
 # __array_ufunc__ should be possible by converting non-scipp arguments to
 # variables. The most difficult part is probably mapping the ufunc to scipp
 # functions.
-for _obj in [Variable, DataArray, Dataset]:
-    setattr(_obj, '__array_ufunc__', None)
-
-from ._scipp import core as tmp_core
-from .utils import get as tmp_get
-
-for cls in [Dataset, tmp_core.Coords, tmp_core.Masks]:
-    setattr(cls, 'get', tmp_get)
-
-del tmp_get
-del tmp_core
+for _cls in (Variable, DataArray, Dataset):
+    setattr(_cls, '__array_ufunc__', None)
+del _cls
+
+from . import _binding
+
+_binding.bind_get()
+for _cls in (Variable, DataArray):
+    _binding.bind_functions_as_methods(
+        _cls, globals(), ('broadcast', 'flatten', 'fold', 'transpose', 'all',
+                          'any', 'mean', 'sum'))
```

Comment:
What about `nansum` and friends?

Reply:
Numpy does not have those as methods. But I have nothing against doing it. Do you think it makes sense?

Reply:
Feels odd to have `sum()` but not `nansum()`?